lib/index.js

```diff
@@ -126,11 +126,10 @@ function* createEverything() {
 
   var adminTestDatabaseClient = pgp(adminTestDatabase);
   yield adminTestDatabaseClient.task(alterPrivileges);
-
-  pgp.end();
```

Comment:
Conceptually I'm trying to completely provision a single template database (I used the `test` one arbitrarily), then clone it entirely for `development` and `production`. So far as I could tell, killing the pool was a prerequisite for cloning, and I didn't readily find an elegant way to do that.